### PR TITLE
feat(cat-voices): account role learn more dialog

### LINKED ISSUE #891: 🛠️ [TASK] : AccountRole LearnMore Dialog
### Summary

Development of generic AccountRole LearnMore Dialog

### Description

Part of #818 

### Requirements

- [ ] static function `show` which will accept **AccountRole** as parameter.
- [ ] show correct content base on provided role in way figma shows

### Design

- [Voter](https://www.figma.com/design/80aDAQgoD0v06DO2Sb76ok/Catalyst--Minimal-Valuable-Experiments-(MVE's)?node-id=5484-36590&m=dev)
- [Main Proposer](https://www.figma.com/design/80aDAQgoD0v06DO2Sb76ok/Catalyst--Minimal-Valuable-Experiments-(MVE's)?node-id=5484-36922&m=dev)
- [dRep](https://www.figma.com/design/80aDAQgoD0v06DO2Sb76ok/Catalyst--Minimal-Valuable-Experiments-(MVE's)?node-id=5484-37254&m=dev)

### PR DESCRIPTION
**Pull request by digitalheartxs**
# Description

Implement account role learn more dialog

## Related Issue(s)

Closes #891 

## Screenshots

Screenshot 2024-10-08 at 14 43 40

Screenshot 2024-10-08 at 14 42 49

Screenshot 2024-10-08 at 14 41 58

## Please confirm the following checks

* [x] My code follows the style guidelines of this project
* [x] I have performed a self-review of my code
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation
* [x] My changes generate no new warnings
* [x] I have added tests that prove my fix is effective or that my feature works
* [x] New and existing unit tests pass locally with my changes
* [x] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

**Comment by dtscalac** [comment]
@digitalheartxs Consider to convert images to webp and import them at different sizes: 1x, 2x, 3x

**Comment by digitalheartxs** [comment]
> @digitalheartxs Consider to convert images to webp and import them at different sizes: 1x, 2x, 3x

I was asking Niels about that, they are temporary images so I decided to don't do additional work for images that are only temporarily.

File: catalyst_voices/lib/widgets/common/link_text.dart [hunk 1]
```diff
@@ -41,13 +41,16 @@ class LinkText extends StatelessWidget {
...
     );
 
-    return GestureDetector(
-      onTap: onTap,
-      child: DefaultTextStyle.merge(
-        style: style,
-        child: Text(
-          data,
-          style: effectiveStyle,
+    return MouseRegion(
```
Comment (digitalheartxs) [inline catalyst_voices/lib/widgets/common/link_text.dart:44]: @damian-molinski the lack of pointer on `LinkText` was a bug, not a requested feature, right?

**Review by dtscalac — APPROVED** [review]

File: catalyst_voices/lib/widgets/common/link_text.dart (hunk 1 above)
Comment (damian-molinski) [reply catalyst_voices/lib/widgets/common/link_text.dart:44]: Correct, didn't think about it

**Review by damian-molinski — APPROVED** [review]

**Pull request merged**